README.md

```diff
@@ -66,6 +66,48 @@ You can also use this script developed by [enoch85](https://github.com/enoch85):
 ```
 #!/bin/bash
 
+# Variables
+DISTRO=$(lsb_release -sd | cut -d ' ' -f 2)
+OS=$(grep -ic "Ubuntu" /etc/issue.net)
```

Comment:
If a user has modified his `issue.net` to something else without the `Ubuntu` string this won't work.
You might try using `uname -v` 
Example output:
```
sander@devserver:~$ uname -v
#121-Ubuntu SMP Tue Oct 10 14:24:03 UTC 2017
```

Reply:
Nice catch!